packages/cli/src/loaders/index.ts

```diff
@@ -109,5 +110,12 @@ export default function createBucketLoader(
       createFlatLoader(),
       createUnlocalizableLoader(),
     );
+    case 'xliff': return composeLoaders(
+      createTextFileLoader(bucketPathPattern),
+      // createPrettierLoader({ parser: 'xliff' }),
+      createXliffLoader(),
+      // createFlatLoader(),
```

Comment:
this should be on by default, unless you intentionally want to keep this disabled